packages/synapse-react-client/src/components/ChallengeSubmission/AvailableEvaluationQueueList.tsx

```diff
@@ -52,37 +52,59 @@ type AvailableEvaluationQueueStaticListProps = Pick<
   AvailableEvaluationQueueListProps,
   'evaluations'
 >
-
 function AvailableEvaluationQueueStaticList(
   props: AvailableEvaluationQueueStaticListProps,
 ) {
   const { evaluations } = props
+  return (
+    <>
+      <Typography variant="body1">Available Evaluation Queues:</Typography>
+      <List dense={true}>
+        {evaluations.map(evaluation => (
+          <ListItem key={evaluation.id}>
+            <TextWithHelpIcon
+              text={evaluation.name!}
+              tooltipMarkdownText={evaluation.submissionInstructionsMessage}
+            />
+          </ListItem>
+        ))}
+      </List>
+    </>
+  )
+}
+
+type AvailableEvaluationQueueCollapsableListProps = Pick<
+  AvailableEvaluationQueueListProps,
+  'evaluations' | 'isSelectable'
+>
+
+function AvailableEvaluationQueueCollapsableList(
+  props: AvailableEvaluationQueueCollapsableListProps,
+) {
+  const { evaluations, isSelectable } = props
   const [showList, setShowList] = useState<boolean>(false)
+  const collapseBound = isSelectable ? 2 : 8
```

Comment:
Could consider storing whether the component should collapse in a boolean for readability -- 

```suggestion
  const nEvaluationsCollapseLimit = isSelectable ? 2 : 8
  const shouldCollapse = evaluations.length >= nEvaluationsCollapseLimit
```